File: 23225/countdi4.cpp
#include <bits/stdc++.h>
using namespace std;
long long factorial(long long n) {
    if (n == 0 || n == 1)
        return 1;
    else
        return n * factorial(n - 1);
}
long long n;
int demuocnt(long long n) {
    int count = 0;
    for (long long i = 2; i <= n; ++i) {
        if (n % i == 0) {
            ++count;
        }
    }
    return count;
}
long long ft;
int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL); cout.tie(NULL);
    freopen("countdi4.inp", "r", stdin);
    freopen("countdi4.out", "w", stdout);
    cin >> n;
    ft=factorial(n);
    cout << demuocnt(ft);
    return 0;
}

File: 23225/countdi4.py
import sys
import math

sys.stdin('countdi4.inp', 'r')
sys.stdout('countdi4.out', 'w')

n=int(input())
s=math.factorial(n)
dem=0
for i in range (1, n):
    if(n%i==0):
        dem+=1
print(dem)